### ISSUE TITLE
Make viewing rights explicit

### ISSUE THREAD

**Issue by bumbleblue** [freedomofpress/securedrop#4091]
## Description
Current language and images suggest that sources communicate with **individual** journalists rather than organizations. The 1-to-group (rather than 1-to-1) communication model can be made more explicit. Example of current explainer below. 

## Quick fix
Make the person on the right a group of three and update language around that.
![sd_diagram width-800-a](https://user-images.githubusercontent.com/14316263/51916155-9f0d5200-23d4-11e9-8527-986da1572f4f.png)

## Better fix
Clear up language around "journalist", "organization", "instances". Make sure buttons say things like "share with [organization name".

**Comment by ninavizz** [comment]
Eileen!! So good to see you here! :)

Agreed, thank you for flagging this! The more eyes and minds on these subtle user-facing bits, the better. 

I created the below, 2 years ago... when I was first getting my head around both Tor and SecureDrop. The 3 little "Newsroom Weebles" should at least be carry-over-able. Maybe the Source Weeble, too (if it doesn't seem like too much of a ripoff of Signal's branding?).


![image](https://user-images.githubusercontent.com/8262612/52158116-da648680-2649-11e9-9957-6d0318eb10a1.png)

I think for communicating how all this works to Source users (whom we've also **_got_** to start calling something more human/clear-sounding to folks unfamiliar with Journalism jargon, than "Source") a clearer illustration with clearer language should absolutely be in order. I wouldn't be surprised if the FPF training team has already come up with something, come to think... (@huertanix?).

Thanks again for flagging. We should also get this (and any improvements) in front of users to get their thoughts; and this would be a terrific tidbit for group user testing.

**Comment by huertanix** [comment]
@ninavizz We don't have anything in terms of illustrating the whole workflow yet. We do have a slide  talking about how Tor hidden services work (technically three hops on each side with a rendezvous point in between):
![tor_hidden_services](https://user-images.githubusercontent.com/72914/52668948-e7ebfd00-2ee2-11e9-8285-e6fd51b510b2.png)

**Comment by ninavizz** [comment]
@huertanix Oh, that's a great step-thru! I don't believe users will be able to grok it w/o a visual, though. I'm happy to make a clearer graphic than what's on the site now, that takes all of these things into consideration. Not this Sprint, tho, maybe next one @eloquence?

Longer-term, it would be nice to chat through some of the Source experience pain-points with @eloquence and @redshiftzero to see how those might get prioritized in the work over the next 6mos. It'd be nice to at a minimum, poke at this and that here and there, as we're all able to—and to have some of those things prioritized.

**Comment by ninavizz** [comment]
Also, I don't believe most users will know what to make of all the IP hops. A potential option is to provide a "Basic" version, and an "Advanced" version; by default, show the "Basic" version, and if a user wants more details then have a clearly discoverable link to an advanced version (which could even just be the workflow chart).

I don't believe most non-technical users (so, the majority of Sources) even know what a MAC address or an IP address really are. I know I sure don't really understand the technical implications of the flow chart you shared—for my tech it's cognitively abstracted as "Squirrel A hands Acorn 01 to Squirrel B..."

**Comment by eloquence** [comment]
@ninavizz Yes, I'm up for working together on a new graphic that better communicates the core concepts, in the near future. The old graphic is outdated in one other important way; it refers to the "Document Interface" which is now called the "Journalist Interface". Perhaps we can aim to create a similar graphic for the Qubes architecture at the same time, so we can use a consistent style to explain both.

It's worth noting that a handful of individual journalists do operate SecureDrops, but I agree that this particular graphic should default to the more common case of multiple journalists.

I suggest that we keep this issue scoped to the documentation aspect, and treat any proposed changes to the source interface (which does not use this graphic) separately.

**Comment by huertanix** [comment]
@ninavizz To be clear, this is one slide which follows an entire deck of slides walking an audience through identifiers in networks, including mac addresses and IP addresses. This is 5th from the last slide, iirc.

**Comment by bumbleblue** [comment]
Hi @ninavizz, how nice to meet here!
I really like the group of journalists in your graphic. I also like how you're trying to add information on Tor hidden services. The thing I would do here is make those two different graphics - for the first case, people only need to see an onion / sth that signifies anonymity. And then, if people want to find out how it works exactly, you can offer a graphic explaining onion routing - or link directly to whichever explainer Tor is using. Otherwise I think the red lines may be a little confusing.

**Comment by eloquence** [comment]
Retitled for clarity. Note that the diagram in the issue body is only used in the [website FAQ](https://securedrop.org/faq/about-securedrop/) at this point; the [overview](https://docs.securedrop.org/en/stable/_images/SecureDrop.png) in the docs uses a different diagram. We should standardize on a single diagram soon, and as part of that, more clearly indicate that the current architecture uses a shared inbox.
